src/components/stopPoster/stopPoster.js

```diff
@@ -79,13 +83,6 @@ class StopPoster extends Component {
         renderQueue.onEmpty(error => !error && this.updateLayout(), { ignore: this });
     }
 
-    componentWillUnmount() {
-        if (this.mapMutationObserver) {
-            this.mapMutationObserver.disconnect();
-            this.mapMutationObserver = null;
```

Comment:
The this.mapMutationObserver seems to be an old thing that was removed previously. No need to keep this around anymore either.